### PR TITLE
Properly url encode

### PR DESCRIPTION
**Pull request by bastbijl**
now im getting this
```➜  /tmp weaver-cli http://localhost:9487 admin admin dataroom-zuidoost snapshot                                                                                      
Exception in thread "main" javax.ws.rs.core.UriBuilderException: java.net.URISyntaxException: Illegal character in query at index 42: http://localhost:9487/snapshot?payload=%7B"target":"dataroom-zuidoost","authToken":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6ImFkbWluIiwiYWRtaW4iOnRydWUsImlhdCI6MTUxMTM1NTE4MiwiZXhwIjoxNTExOTU5OTgyfQ.J18upPaiXjp9b7hEbbNpwYNrjqSJvVCiy6WddeOiw7I"%7D
	at com.sun.jersey.api.uri.UriBuilderImpl.createURI(UriBuilderImpl.java:723)
	at com.sun.jersey.api.uri.UriBuilderImpl._build(UriBuilderImpl.java:651)
	at com.sun.jersey.api.uri.UriBuilderImpl.build(UriBuilderImpl.java:641)
	at com.sun.jersey.api.client.WebResource.<init>(WebResource.java:88)
	at com.sun.jersey.api.client.WebResource.queryParam(WebResource.java:443)
	at com.weaverplatform.sdk.Weaver.send(Weaver.java:97)
	at com.weaverplatform.sdk.Weaver.sendAuthenticated(Weaver.java:87)
	at com.weaverplatform.sdk.Weaver.sendProject(Weaver.java:79)
	at com.weaverplatform.sdk.Weaver.getSnapshot(Weaver.java:212)
	at com.weaverplatform.Main.performOperation(Main.java:47)
	at com.weaverplatform.Main.main(Main.java:81)
Caused by: java.net.URISyntaxException: Illegal character in query at index 42: http://localhost:9487/snapshot?payload=%7B"target":"dataroom-zuidoost","authToken":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6ImFkbWluIiwiYWRtaW4iOnRydWUsImlhdCI6MTUxMTM1NTE4MiwiZXhwIjoxNTExOTU5OTgyfQ.J18upPaiXjp9b7hEbbNpwYNrjqSJvVCiy6WddeOiw7I"%7D
	at java.net.URI$Parser.fail(URI.java:2848)
	at java.net.URI$Parser.checkChars(URI.java:3021)
	at java.net.URI$Parser.parseHierarchical(URI.java:3111)
	at java.net.URI$Parser.parse(URI.java:3053)
	at java.net.URI.<init>(URI.java:588)
	at com.sun.jersey.api.uri.UriBuilderImpl.createURI(UriBuilderImpl.java:721)
	... 10 more
```

### CODE REVIEW

**Comment by malamili** [comment]
I dont get your comment..is this PR a WIP?

File: src/main/java/com/weaverplatform/sdk/Weaver.java [hunk 1]
```diff
@@ -94,7 +96,12 @@ public ClientResponse send(JsonObject payload, String target, OP op) {
     ClientResponse soapResponse;
     switch (op) {
       case get:
-        webResource = webResource.queryParam("payload", converter.toJson(payload));
+        try {
+          String json = URLEncoder.encode(converter.toJson(payload), "UTF-8");
+          webResource = webResource.queryParam("payload", json);
+        } catch (UnsupportedEncodingException e) {
+          e.printStackTrace();
```
Comment (gvdent) [inline src/main/java/com/weaverplatform/sdk/Weaver.java]: Throw a runtime exception here, even though it should never happen. 

It is a bad habit to just catch exceptions and move on as if they didn't happen. Either handle it, or rethrow.

File: src/main/java/com/weaverplatform/sdk/Weaver.java [hunk 2]
```diff
@@ -209,7 +216,9 @@ public JsonObject createRelation(String from, String key, String to, String id)
   }
 
   public String getSnapshot() {
-    ClientResponse soapResponse = sendProject(new JsonObject(), "/snapshot", OP.get);
+    JsonObject payload = new JsonObject();
+    payload.addProperty("zipped", true);
+    ClientResponse soapResponse = sendProject(payload, "/snapshot", OP.get);
```
Comment (gvdent) [inline src/main/java/com/weaverplatform/sdk/Weaver.java]: This.... has nothing to do with url encoding does it?

**Comment by gvdent** [comment]
@malamili I suspect the comment to be the description of the issue this fixes :)

**Pull request merged**